src/components/AlertBar.tsx

```diff
@@ -16,7 +16,6 @@ const AlertBar: React.FC<Props> = ({ snackbarService }) => {
     <Snackbar
       anchorOrigin={{ vertical: "bottom", horizontal: "left" }}
       open={snackbarState?.matches("visible")}
-      autoHideDuration={3000}
```

Comment:
@nzwsch Please restore this default and `autoHideDuration` to this PR.  We would like to keep this default.

Reply:
@kevinold My bad. I thought the snackbarMachine would handle invisible.